packages/svelte-urql/src/subscriptionStore.ts

```diff
@@ -0,0 +1,57 @@
+import { pipe, subscribe } from 'wonka';
+import type { OperationContext } from '@urql/core';
+import { derived, writable } from 'svelte/store';
+import { createRequest } from '@urql/core';
+import type {
+  AnnotatedOperationResult,
+  UrqlStore,
+  UrqlStoreArgs,
+} from './common';
+import { defaultBaseResult, fetchProcess } from './common';
+
+/**
+ * Create a Svelte store for an [Urql subscription](https://formidable.com/open-source/urql/docs/api/core/#clientexecutemutation) using [Wonka](https://wonka.kitten.sh/)
+ */
+export function subscriptionStore<Data, Variables extends object = {}>(
+  args: UrqlStoreArgs<Data, Variables>
```

Comment:
I noticed that react binding allows pausing the subscription. https://github.com/FormidableLabs/urql/blob/f86763cc565f0c36b948ce9017da0c7c2c302d58/packages/react-urql/src/hooks/useSubscription.ts#L21

I have not implemented that yet.